lambdas/tests/unit/enums/test_edge_presign_values.py

```diff
@@ -1,16 +1,30 @@
-# test_enums.py
-
 from enums.lambda_error import LambdaError
 
 ENV = "test"
+EXPECTED_ENVIRONMENT = ENV
 
 TABLE_NAME = "CloudFrontEdgeReference"
```

Comment:
Table names are stored in `tests/unit/conftest.py` can this be placed there instead? Where is should also be set as an env var in the `set_env` function, that way we have access to it across the codebase should future tests/services require access to this name